src/index.html

```diff
@@ -18,5 +22,94 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
+        />
+
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="apple"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
```

Comment:
Don't use spaces in <a> tag's `href` property. Anchor links start with the `#` symbol. Change `href="apple"` to `href="#apple"`.